packages/liveblocks-core/src/protocol/ServerMsg.ts

```diff
@@ -275,6 +275,11 @@ export type RoomStateServerMsg<U extends BaseUserMeta> = {
   readonly users: {
     readonly [otherActor: number]: U & { scopes: string[] };
   };
+
+  /**
+   * Optional metadata sent from the server to the client.
+   */
+  readonly meta?: Record<string, unknown>;
```

Comment:
Could we make the `unknown` more specific? At the very least it should be `Record<string, Json>` (because it comes from a JSON message), but maybe we can be even more precise, like `Record<string, string | string[]>` if we know the server will only ever send that?